src/Content/Type/Sniffer.php

```diff
@@ -25,17 +27,26 @@ class Sniffer
     /**
      * File object
      *
-     * @var \SimplePie\File
+     * @var File
      */
     public $file;
 
     /**
      * Create an instance of the class with the input file
      *
-     * @param File $file Input file
+     * @param File|Response $file Input file
      */
-    public function __construct(File $file)
+    public function __construct(/* File */ $file)
```

Comment:
Removing the type hint is not a breaking change because we have not released 1.9.0 yet. ([refs](https://github.com/simplepie/simplepie/pull/777/files#r1108580819))

Also the type hint in the docblock was wrong, see https://github.com/simplepie/simplepie/blob/1.8.0/src/Content/Type/Sniffer.php#L71-L79